src/main/java/com/openshift/jenkins/plugins/pipeline/ImageStreamRevisionState.java

```diff
@@ -9,7 +9,7 @@
 	public ImageStreamRevisionState(String commitId) {
 		if (commitId == null)
 			throw new IllegalArgumentException("nulls not allowed");
-		this.commitId = commitId;
+		this.commitId = commitId != null ? commitId.trim() : null;
```

Comment:
well, in this case, null is checked and exception thrown before hand in the line above ... we can just do commitId.trim() here.